mkdocs.yml

```
@@ -0,0 +1,93 @@
+            show_root_heading: true
+            heading_level: 2
+            members_order: source
+            docstring_style: google
+            show_if_no_docstring: false
+            filters: ["!^_"]
+            merge_init_into_class: true
+
+nav:
+  - Home: index.md
+  - Getting Started:
+    - Installation: getting-started/installation.md
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)